docs/source/roles.md

```diff
@@ -92,7 +92,8 @@ The IT professionals and software engineers who will be responsible for developi
 ## Governance Roles
 
 Roles that uphold the governance of TREs.
-Such governance responsibilities typically involve establishing policies and procedures to ensure the responsible use of data, protecting the privacy and confidentiality of research participants, and promoting transparency and accountability in research activities.
+Such governance responsibilities typically involve establishing policies and procedures to ensure the responsible use of data.
+This should protect the privacy and confidentiality of research participants, as well as promote transparency and accountability in research activities.
```

Comment:
I'm not sure I see the need to split this sentence from the preceding one. Depends a bit. Do these clauses belong to "governance responsibilities" or to "responsible use of data"? Splitting it like this makes it read to me like both of these things follow from responsible use of data, which is partly true, but I think promoting transparency and accountability in research activities is distinct from responsible use of data. Responsible use of data might include transparency and accountability, but I don't think it exactly promotes i.